source/sdk/math.ooc

```diff
@@ -358,6 +318,9 @@ extend Float {
 }
 
 extend LDouble {
+	pi ::= static 3.14159_26535_89793_23846_26433_83279 as This
+	e ::= static 2.718281828459045235360287471352662497757247093699959574966 as This
```

Comment:
I find nothing in the documentation about how to specify an `LDouble`, so I don't know.